modules/core/02-client/types/store.go

```diff
@@ -30,3 +30,8 @@ func (s storeProvider) ClientStore(ctx sdk.Context, clientID string) storetypes.
 	clientPrefix := []byte(fmt.Sprintf("%s/%s/", host.KeyClientStorePrefix, clientID))
 	return prefix.NewStore(ctx.KVStore(s.storeKey), clientPrefix)
 }
+
+// ModuleStore returns the 02-client module store
+func (s storeProvider) ModuleStore(ctx sdk.Context, clientType string) storetypes.KVStore {
+	return prefix.NewStore(ctx.KVStore(s.storeKey), host.KeyClientStorePrefix)
+}
```

Comment:
I think we should be using `host.PrefixedClientStoreKey([]byte(clientType))` here :) 

ref:

```go
// PrefixedClientStoreKey returns a key which can be used for prefixed
// key store iteration. The prefix may be a clientType, clientID, or any
// valid key prefix which may be concatenated with the client store constant.
func PrefixedClientStoreKey(prefix []byte) []byte {
	return []byte(PrefixedClientStorePath(prefix))
}

```

Reply:
Good point. And then the `clientType` argument is actually used. 😄